content/blog/data-analytic-with-pyspark-using-hpe-ezmeral-container-platform.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Data Analytic with pySpark Using HPE Ezmeral Container Platform
+date: 2021-09-07T02:12:15.242Z
+author: Cenz Wong
+authorimage: https://avatars.githubusercontent.com/u/44856918?v=4
+thumbnailimage: ""
+tags:
+  - hpe-ezmeral-container-platform
+  - hpe-ezmeral-data-fabric
+---
+
+
+PySpark is an interface for Apache Spark in Python. [Apache Spark](https://spark.apache.org/) is a unified analytics engine for big data processing. It allows developers to perform data processing on files in distributed filesystem, like Hadoop distributed filesystem or HPE Ezmeral Data Fabric (formerly known as MapR-XD). Setting up Spark environment is always a pain for data scientists. HPE Ezmeral Container Platorm is here for making your life easier. You can run Spark with Apache Livy and Spark operator or directly run Spark job with pySpark module. In this blog, I will be focusing on running Spark jobs using pySpark module. For those who want to squeeze the best performance of Spark, visit this [blog](https://developer.hpe.com/blog/on-premise-adventures-how-to-build-an-apache-spark-lab-on-kubernetes/).
+
+
+
+## Preparing the Notebook Cluster
+
+
+First, we have to prepare our favorite Jupyter environment. Inside a MLOps tenant, navigate to Notebooks tab. You will see a Jupyter KubeDirector app prepared for you. After clicking the "Launch" button, you will need to configure the compute resource needed.
```

Comment:
Not related to workshop-on-demand. My tone was like to be a tour guide, showing the world to the audience. :-) I will change those line and add more sentence later.

Reply:
Hi @cenzwong 
I took the liberty to add a bit of context to your blog post in the introductory paragraph. Feel free to adjust as appropriate or use it as is.
Thanks
Denis

Reply:
The Environment should be name as Jupyter Lab, which is the next Gen of Jupyter Notebook

Reply:
(I just find out that the kubedirector app image has the name of Jupyter Notebook .... So maybe it should be fine to use those term

Reply:
Hi @cenzwong 

Exactly, ECP kubedirector name is Jupyter Notebook, hence I had suggested Jupyter Notebook in the text.

One typo Apache Livy to make REST API calls to Spark Operator and Spark operator, or you can directly...
-->Delete the "and Spark operator"

Maybe I would suggest a line break after ...with PySpark module so the it can be read like this:
...or you can directly run a Spark job against the Spark Operator with the PySpark module. 

In this post, I will focus...